tools/actions/commit-queue.sh

```diff
@@ -64,30 +76,25 @@ for pr in "$@"; do
 
   # TODO(mmarchini): workaround for ncu not returning the expected status code,
   # if the "Landed in..." message was not on the output we assume land failed
-  if ! tail -n 10 output | grep '. Post "Landed in .*/pull/'"${pr}"; then
-    gitHubCurl "$(labelsUrl "$pr")" POST --data '{"labels": ["'"${COMMIT_QUEUE_FAILED_LABEL}"'"]}'
-
-    # shellcheck disable=SC2154
-    cqurl="${GITHUB_SERVER_URL}/${OWNER}/${REPOSITORY}/actions/runs/${GITHUB_RUN_ID}"
-    jq -n --arg content "<details><summary>Commit Queue failed</summary><pre>$(cat output)</pre><a href='$cqurl'>$cqurl</a></details>" '{body: $content}' > output.json
-    cat output.json
-
-    gitHubCurl "$(commentsUrl "$pr")" POST --data @output.json
-
-    rm output output.json
+  if ! grep -q '. Post "Landed in .*/pull/'"${pr}" output; then
+    commit_queue_failed "$pr"
     # If `git node land --abort` fails, we're in unknown state. Better to stop
     # the script here, current PR was removed from the queue so it shouldn't
     # interfer again in the future
```

Comment:
```suggestion
    # interfere again in the future
```

Reply:
This can be fixed in another PR, this PR may need an immediate revert if it breaks the CQ.